js/script.js

```diff
@@ -196,3 +194,40 @@ const displayModal = (event) => {
 workButtons.forEach((btn) => {
   btn.addEventListener('click', displayModal);
 });
+
+// Validating mail form
+const firstName = document.querySelector('#first-name');
+const lastName = document.querySelector('#last-name');
+const fullName = document.querySelector('#name');
+
+function validateEmail(inputElement) {
+  const username = inputElement.value.split('@')[0];
+  const emailRegex = /^[a-z_.\-|1-9]+$/;
+  return emailRegex.test(username);
+}
```

Comment:
Your lower case is validation is working as expected but only when the upper case is before the @ 😵 If the uppercase is after the @ the form submits 😥 Kindly make the necessary changes so that the validation is performed on the entire string